pkg/dao/rpms_test.go

```diff
@@ -1062,8 +1062,9 @@ func (s *RpmSuite) TestDetectRpms() {
 				},
 			},
 			expected: api.DetectRpmsResponse{
-				Found:   []string{"demo-package", "test-package"},
-				Missing: []string{"fake-package"},
+				Found:         []string{"demo-package", "test-package"},
+				FoundPackages: []api.SearchRpmResponse{{PackageName: "demo-package", Summary: "demo-package Epoch 1"}, {PackageName: "test-package", Summary: "test-package Epoch 1"}},
```

Comment:
ohp missed the unit test failure, i think these summaries should just be `demo-package Epoch 0` and `test-package Epoch 0`